### PR TITLE
RIA-8271 Print and send hearing requirements form

### PR DESCRIPTION
**Pull request by javierhmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-8271

### Change description ###

Print and send hearing requirements form WA-TASK

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/resources/wa-task-configuration-ia-asylum.dmn [hunk 1]
```diff
@@ -202,7 +202,7 @@
                        "reviewAppealSetAsideUnderRule32",
                        "reviewRemittedAppeal", "processFeeRefund",
                        "reviewDraftAppeal",
-                       "printAndSendHoBundle"
+                       "printAndSendHoBundle", "printAndSendHearingRequirements"
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-configuration-ia-asylum.dmn:205]: For easier to rebase in the future, can you add this on the next line? Follow the format above

File: src/main/resources/wa-task-configuration-ia-asylum.dmn [hunk 2]
```diff
@@ -673,7 +673,7 @@
 "uploadHearingRecording",
 "postHearingAttendeesDurationAndRecording",
 "decideAnFTPA",
-"printAndSendHoBundle"</text>
+"printAndSendHoBundle", "printAndSendHearingRequirements"</text>
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-configuration-ia-asylum.dmn:676]: For easier to rebase in the future, can you add this on the next line? Follow the format above

File: src/main/resources/wa-task-initiation-ia-asylum.dmn [hunk 3]
```diff
@@ -3748,6 +3748,65 @@
           <text>"caseProgression"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0pjq6cy">
+        <inputEntry id="UnaryTests_0mpynjv">
+          <text>"requestHearingRequirements"</text>
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-initiation-ia-asylum.dmn:3753]: Think this is the wrong event id, and now redundant. Can you please check.

**Review by ThomasKKC — CHANGES_REQUESTED** [review]

File: src/main/resources/wa-task-initiation-ia-asylum.dmn [hunk 4]
```diff
@@ -3748,6 +3748,65 @@
...
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0da6scr">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_055shpl">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0cbrnnv">
+          <text>true</text>
```
Comment (ThomasKKC) [inline src/main/resources/wa-task-initiation-ia-asylum.dmn:3795]: Can you make sure this is published on the event?

**Review by ThomasKKC — CHANGES_REQUESTED** [review]

**Review by ThomasKKC — APPROVED** [review]

**Review by colmlynch60 — COMMENTED** [review]
LGTM

**Pull request merged**